### ISSUE TITLE
 Add Feature to Assign a Day for Grocery Lists

### ISSUE THREAD

**Issue by Avinash-Codes** [yuvrajsinghgmx/ShopSmart#84]
Description:
This feature will allow users to specify a particular day for which the grocery list is intended. This will help users better plan their shopping trips based on the day they wish to complete the list.

Feature Details:
Day Selection: Users should be able to assign a specific day (e.g., Monday, Tuesday) to each grocery list when creating list.

**Comment by Avinash-Codes** [comment]
Please Assign me this

**Comment by yuvrajsinghgmx** [comment]
Integrate this for the UpComing Page

**Comment by Avinash-Codes** [comment]
okh it may take some time because i am having exam this week

**Comment by yuvrajsinghgmx** [comment]
> okh it may take some time because i am having exam this week

Okay No Worries.

**Comment by yuvrajsinghgmx** [comment]
@Avinash-Codes You can continue on your previous approach and please refer to #4 once and ask if you have any queries.

**Comment by Avinash-Codes** [comment]
> Hi @Avinash-Codes You can continue on your previous approach and please refer to #4 once and ask if you have any queries.

I need a few clarifications: Are you building a shopping app, or are you creating a to-do list for shopping to help people remember things they need to buy? I ask this because the screenshots you added seem to indicate that it's a shopping app, which is different from what you're describing in the repository

**Comment by yuvrajsinghgmx** [comment]
You are absolutely right @Avinash-Codes the list page on which you have worked will act as to-do list and the home page will serve as a searching from various e-commerce sites the Buy button will take the user on respective e-commerce site.
I will modify the description soon and please let me know if you have further queries.

**Comment by Avinash-Codes** [comment]
> You are absolutely right @Avinash-Codes the list page on which you have worked will act as to-do list and the home page will serve as a searching from various e-commerce sites the Buy button will take the user on respective e-commerce site. I will modify the description soon and please let me know if you have further queries.

ok fine

**Comment by Avinash-Codes** [comment]
Done With this
